.github/workflows/appsignals-e2e-ec2-test.yml

```diff
@@ -171,6 +171,9 @@ jobs:
           curl -S -s http://${{ env.MAIN_SERVICE_ENDPOINT }}/remote-service?ip=${{ env.REMOTE_SERVICE_IP }}/
           curl -S -s http://${{ env.MAIN_SERVICE_ENDPOINT }}/client-call/
 
+      - name: Initiate Gradlew Daemon
+        run: ./gradlew
```

Comment:
Regardless of the improvement, can we wrap this step in a retry?

Reply:
Done